joshuaCarpentier/quiz/queue1.js

```diff
@@ -0,0 +1,70 @@
+// Problem Statement
+// Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+
+
+  /**
+   * 
+   * Input: Array of positive numbers & a positive num k
+   * Output: Sum
+   *    
+   * find sum of any contiguous subarray compare to k 
+   * 
+   * Naive Approach
+   * --------------- 
+   * Variables: 
+   * 
+   * 
+   */
+   console.log(`\n ------- Test Case -------- `);
+
+const maxSum = (arr, k) =>{
+    console.log(`${arr}`)
+    if(!arr.length) return k
+    
+
+
+    let totalSum =0;
+
+    for(let i=0; i<arr.length; i++){
+        // if(k === 0) {
+        //     //check if array has 0
+        // }
+        //  if(arr.length == 1)  arr[i] 
+           
+        console.log(`TotalSum After: ${totalSum}`)
+
+        totalSum += arr[i];
+        console.log(`TotalSum ${totalSum}`)
+
+        if(totalSum === k ){
```

Comment:
k is the size of the subarray, so we should be comparing a sum of an array of size k to a max sum and re-assigning max sum if the current sum is larger. K is a subarray size, not a sum.